share/spack/gitlab/cloud_pipelines/stacks/e4s-mac/spack.yaml

```diff
@@ -39,39 +39,15 @@ spack:
 
   mirrors: { "mirror": "s3://spack-binaries/develop/e4s-mac" }
 
-  gitlab-ci:
-
-    script:
-      - tmp="$(mktemp -d)"; export SPACK_USER_CONFIG_PATH="$tmp"; export SPACK_USER_CACHE_PATH="$tmp"
-      - . "./share/spack/setup-env.sh"
-      - spack --version
-      - spack arch
-      - cd ${SPACK_CONCRETE_ENV_DIR}
-      - spack env activate --without-view .
-      - spack config add "config:install_tree:projections:${SPACK_JOB_SPEC_PKG_NAME}:'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'"
-      - mkdir -p ${SPACK_ARTIFACTS_ROOT}/user_data
-      - spack --color=always --backtrace ci rebuild > >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_out.txt) 2> >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_err.txt >&2)
-
-    match_behavior: first
-    mappings:
-      - match: ['os=monterey']
-        runner-attributes:
-          tags:
-          - lambda
-
-    broken-specs-url: "s3://spack-binaries/broken-specs"
-
-    service-job-attributes:
-      before_script:
-      - export SPACK_USER_CACHE_PATH=$(pwd)/.spack-user-cache
-      - export SPACK_USER_CONFIG_PATH=$(pwd)/.spack-user-config
-      - . "./share/spack/setup-env.sh"
-      - spack --version
-      tags:
-      - lambda
-
-  cdash:
-    build-group: E4S Mac
-    url: https://cdash.spack.io
-    project: Spack Testing
-    site: Cloud Gitlab Infrastructure
+  ci:
+    pipeline-gen:
+    - cleanup-job:
```

Comment:
Is this cleanup-job configured because the e4s-mac stack is using the temporary storage feature?  If so, I think both could just be removed.

Reply:
I put it in to keep it consistent, but it looks like all of these macos jobs are disabled at the .gitlab-ci.yml level. I will leave removing it to @eugeneswalker since this is an E4S stack.